src/CMakeLists.txt

```diff
@@ -86,6 +86,15 @@ else (ENABLE_DASSERT)
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
 endif (ENABLE_DASSERT)
 
+# Turns on ADDB-based TDSB wrappers.
+# When this flag is disabled, KVSFS will be used the no-oop
+# wrappers.
+# When this flag is enabled, the utils module has to be
+# be compiled with this flag enabled otherwise some of
+# the function calls will be undefined.
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DENABLE_TSDB_ADDB")
```

Comment:
Does this means we are enabling the flag here ?

Reply:
Yes these flags are enabled as of now for testing and further integration work with decoder, but we can turn it off anytime if needed.

Reply:
Addressed this review comment in all the PRs of this story, please check.